
m68k took a while to get a linker script for build-id, so don't use it.

  LD      .tmp_vmlinux1
m68k-linux-gnu-ld: .tmp_vmlinux1: Not enough room for program headers, try linking with -N
m68k-linux-gnu-ld: final link failed: Bad value
make: *** [.tmp_vmlinux1] Error 1


--- linux-2.6.26/Makefile.orig	2010-08-25 01:06:01.000000000 +1000
+++ linux-2.6.26/Makefile	2010-08-25 12:40:09.000000000 +1000
@@ -563,7 +563,7 @@
 
 # Use --build-id when available.
 LDFLAGS_BUILD_ID = $(patsubst -Wl$(comma)%,%,\
-			      $(call ld-option, -Wl$(comma)--build-id,))
+			      $(call ld-option, -Wl$(comma)--build-id=none,))
 LDFLAGS_MODULE += $(LDFLAGS_BUILD_ID)
 LDFLAGS_vmlinux += $(LDFLAGS_BUILD_ID)
 
